athena-dynamodb/pom.xml

```diff
@@ -80,6 +80,19 @@
             <version>${log4j2Version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>athena-federation-integ-test</artifactId>
+            <version>1.0</version>
```

Comment:
all our modules should use the same convention for version year.week.num

Reply:
Replaced version 1.0 with 2020.46.1